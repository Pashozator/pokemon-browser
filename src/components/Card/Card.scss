.card {
  background: #fff;
  text-align: left;
  box-shadow: 0 3px 4px 0 rgba(0, 0, 0, .1);
  font-weight: 600;
  border: none;
}
