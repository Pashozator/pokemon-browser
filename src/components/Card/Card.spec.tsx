import React from 'react'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Card from './Card';

describe('Card component', () => {
	it('should render', () => {
		const wrapper = shallow(<Card>Card test</Card>);

		expect(wrapper.exists()).toBeTruthy();
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
