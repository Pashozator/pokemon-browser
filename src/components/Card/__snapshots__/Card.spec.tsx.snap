// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render 1`] = `
<div
  className="card "
>
  Card test
</div>
`;
